configs/gtp/gtp-adv600-vm1-s.cfg

```diff
@@ -0,0 +1,9 @@
+@include gtp-v1.cfg
```

Comment:
should we move our existing configs `gtp-amcts.cfg`, `gtp-base.cfg`, `gtp-raw.cfg` into `configs/gtp`? Or just rename `configs/gtp` to `configs/sabaki`?
I'm also wondering whether `gtp-v1.cfg` should import `gtp-base.cfg` and make modifications from there

Reply:
Moved and refactored configs.